header.php

```diff
@@ -52,11 +52,13 @@
 
   $logo_id = IGV_get_option('_igv_social_options', '_igv_metadata_logo_id');
```

Comment:
microdata is a type of metadata, might not exactly be called that. check the `schema-org.php` partial for implimentation